src/workers/parser/tests/fixtures/async.js

```diff
@@ -8,3 +8,8 @@ async function stuff() {
   await foo(1);
   await foo(2);
 }
+
+function* bar() {
```

Comment:
this should be moved into its own file (called "generators")

reason being:
`yield` isnt asynchronous. its a generator. generators are processes that can be paused and resumed. it is the building block of the async/await keyword, but is not itself asynchronous.

here is a good write up, in case you are curious in reading further :) http://exploringjs.com/es6/ch_generators.html